include/seqan3/alignment/matrix/alignment_trace_algorithms.hpp

```diff
@@ -15,6 +15,7 @@
 #include <deque>
 #include <vector>
 
+#include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp>
 #include <seqan3/alignment/matrix/alignment_coordinate.hpp>
 #include <seqan3/alignment/matrix/alignment_trace_matrix.hpp>
 #include <seqan3/alphabet/gap/gapped.hpp>
```

Comment:
can you remove this now?

Reply:
No, because I need the `insert_gap` function...

Reply:
ah yes, it is defined though the concept, so gapped can be removed. Thanks!